library/docs/docs.css

```diff
@@ -0,0 +1,94 @@
+.docs{
+   display: flex;
+   flex-wrap: wrap;
+   padding-left: 1rem;
+   padding-right: 1rem;
+   margin: 1rem auto;
+}
+
+aside {
+   flex-basis:20rem;
+   background-color:var(--Turquoisecolor);
+   height:50rem;
```

Comment:
add some padding so that the content is not sticking to the box

Reply:
Amlan you can use prettier code extension for vscode, it auto-formats these small spacing which we generally miss out.

Reply:
thanks @saurabhan  i will add padding here.

Reply:
thanks for the suggestion @bhushannaikaroskar I will use this extension.